packages/web3-utils/types/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I think it should be possible to have a global tsconfig.json instead of a config for each package.

Reply:
yeah this will not work as `dtslint` needs the `tsconfig.json` within the `types` folder to run..

 `Error: Need a 'tsconfig.json' file in C:\Users\Josh Stevens\repos\web3.js\packages\web3-utils\types` i will look if they can support a global `tsconfig.json`

Reply:
https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#configuration-inheritance-with-extends

I think this should work :)

Reply:
as spoken offline `dtslint` do not support the extends property at the moment - I am going to do a PR for them so we can support this but for now i will mark this as resolved